src/lib/schematics/table/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.html

```diff
@@ -1,21 +1,20 @@
 <div class="mat-elevation-z8">
-  <mat-table #table [dataSource]="dataSource" matSort aria-label="Elements">
-
+  <table mat-table #table [dataSource]="dataSource" matSort aria-label="Elements">
```

Comment:
Didn't @andrewseguin already change this?

Reply:
He changed the one in the component, not the one in the html file.